datacats/cli/create.py

```diff
@@ -20,7 +20,7 @@ def create(opts):
     """Create a new environment
 
 Usage:
-  datacats create [-bin] [--ckan=CKAN_VERSION] ENVIRONMENT_DIR [PORT]
+  datacats create [-bin] [-c NAME] [--ckan=CKAN_VERSION] ENVIRONMENT_DIR [PORT]
```

Comment:
I'm torn on this.

Should we allow the user to choose a child name on create or just force them to create "primary" and use init to add children. Can you think of a reason to create a new environment and custom child name at the same time?